lib/error.js

```diff
@@ -38,7 +38,7 @@ function isResumableError(error) {
 
   return !(
     GET_MORE_NON_RESUMABLE_CODES.has(error.code) ||
-    error.hasErrorLabel('NonRetryableChangeStreamError')
+    error.hasErrorLabel('NonResumableChangeStreamError')
```

Comment:
This seems to have been based on an error in the spec - the spec prose says to check for `NonRetryableChangeStreamError` but the actual spec tests check for  `NonResumableChangeStreamError`.

See [line 50](https://github.com/mongodb/specifications/pull/528/files#diff-875f237661962b4ca9b2e58d97449831R50) vs [line 109](https://github.com/mongodb/specifications/pull/528/files#diff-3495dd67a317bc39589c3cb913399166R109).

Reply:
My reasoning was this isn't related to 4.4 support, so it should go into 3.5 as well.

Reply:
I don't think your assumption is incorrect, but we need make tradeoffs here. Now that we are in the process of developing 3.6 (and indeed it has been beta'd), we expect 3.5 to be just a "maintenance" branch. While the changes represented here fall into maintenance (bug fixes), I think they are larger in scope than just a bug fix. We expect users to starting using 3.6 immediately when it's released, so I think we should rebase these changes there to ease your development. 

Having said that, if it's not too difficult to backport those other changes we depend upon then I don't feel strongly about fixing this for 3.5 as well. I just don't want to spend a ton of time on the fix.

Reply:
> so I think we should rebase these changes there to ease your development.

> Having said that, if it's not too difficult to backport those other changes we depend upon then I don't feel strongly about fixing this for 3.5 as well. I just don't want to spend a ton of time on the fix.

I think it should be quite straightforward to port this between 3.5/3.6. The tests are passing, so I think it's safe to keep this in 3.5 and then port it forward; but I can rebase to 3.6, just LMK.

Reply:
iiuc checking for this error label is just incorrect behavior now though, right? Like this isn't specific to 4.4 or not, we're just no longer supposed to check for this ([NODE-2565](https://jira.mongodb.org/browse/NODE-2565))